src/database/mysql/mysqlcmd.pm

```diff
@@ -307,13 +307,13 @@ __END__
 
 =head1 NAME
 
-mysqlcmd global
+MySQL command global
 
 =head1 SYNOPSIS
 
-mysqlcmd class
+MySQL command class
 
-=head1 MYSQLCMD OPTIONS
+=head1 MYSQL COMMAND OPTIONS
```

Comment:
This change will beak the help option (l. 56) 
```perl
$options{options}->add_help(package => __PACKAGE__, sections => 'MYSQLCMD OPTIONS', once => 1);
```

Reply:
Fixed, thanks 👍🏾